lib/td/client/api.rb

```diff
@@ -74,6 +74,8 @@ def initialize(apikey, opts={})
     @retry_post_requests = opts[:retry_post_requests] || false
     @retry_delay = opts[:retry_delay] || 5
     @max_cumul_retry_delay = opts[:max_cumul_retry_delay] || 600
+    @cert_path = opts[:cert_path]
+    @skip_cert_verify = opts[:skip_cert_verify] || false
```

Comment:
We can merge this into one option, for example:
```suggestion
    @verify_cert = opts[:verify_cert] # true/false or file path
```

Then later:
```
    if @verify_cert == false
        client.ssl_config.verify_mode = OpenSSL::SSL::VERIFY_NONE
    elsif @verify_cert.is_a? String
        client.ssl_config.add_trust_ca(@verify_cert)
    end
```

Reply:
@tung-vu-td Thank you.
But I don't think we should merge these options because each has each own purpose.

Reply:
I kinda disagree. Unless you expose client.ssl_config.verify_mode so  user can specify all mode, if we just have @skip_cert_verify as true/false, it doesn't add much functionality as an option.

for example, trino does merge these options:
https://github.com/trinodb/trino-python-client#ssl-verification

Reply:
Thanks.
I know your suggestion is legit. But there is sometimes that kind of inconsistent request.
So I just want to leave that decision to users.
Anyway, I will follow your request.

Reply:
Thank you ver much @mickeey2525 :bow: